### ISSUE TITLE
 Beginners guide for making changes (branch, edit, pull request)

### ISSUE THREAD

**Issue by eppfel** [nextcloud/documentation#180]
Something like this https://github.com/nextcloud/nextcloud.com/issues/137#issuecomment-247805299 is missing from the current docs:
https://docs.nextcloud.com/server/10/developer_manual/bugtracker/codereviews.html

@MariusBluem You want to add it?

**Comment by MariusBluem** [comment]
Hm ... I've also thought about this 😁 ...

**Comment by MariusBluem** [comment]
I will create a step-by-step guide for some basic operations for beginners! :) ...for more complex and advanced operations - GitHub Help or Git Tutorials should be fine 😉

**Comment by eppfel** [comment]
I think the documentation is beast right now. Of course we don't to have PR that are not tested and all, but still it would be nice to have an overview of the process, right?

**Comment by hagengraf** [comment]
@MariusBluem This is such a wonderful idea. I recently gave up. It would be wonderful to have a "documentation" how to contribute to the documentation and how the documentation is structured and why it is like that and what to do if there is a new topic/feature and all these little things - Love it!

**Comment by MariusBluem** [comment]
@hagengraf This reminds me of the Hackweek 😅 **How is GitHub working and how do we use it here at Nextcloud?** .. This is the question i want to answer with this here :) ...`master`-branch, `stableX`-branch ... What does this mean in Documentation ... What does this mean for apps and the server ... I will start writing soon :)

**Comment by MariusBluem** [comment]
...i'll keep in mind that we should consider about the technical future of documentation here ... before we start to document things that might be outdated in a few months :grin: @hagengraf

**Comment by Chartman123** [comment]
Is this still an issue? There is some basic information about how to contribute in Readme.md...

**Comment by skjnldsv** [comment]
This should be covered by https://nextcloud.com/developer/ which gives a lot of hellp on how to get started :)
